rpc/grpc/core/proto/rpc.proto

```diff
@@ -819,19 +846,21 @@ message GetMetricsResponseMessage{
   ConnectionMetrics connectionMetrics = 12;
   BandwidthMetrics bandwidthMetrics = 13;
   ConsensusMetrics consensusMetrics = 14;
+  StorageMetrics storageMetrics = 15;
   RPCError error = 1000;
 }
 
 message GetServerInfoRequestMessage{
 }
 
 message GetServerInfoResponseMessage{
-  repeated uint32 rpcApiVersion = 1; // Expecting exactly 4 elements
-  string serverVersion = 2;
-  string networkId = 3;
-  bool hasUtxoIndex = 4;
-  bool isSynced = 5;
-  uint64 virtualDaaScore = 6;
+  uint32 rpcApiVersion = 1;
+  uint32 rpcApiRevision = 2;
```

Comment:
It's a breaking changes. Why not adding them to the end?

Reply:
Yes, it is a breaking change for gRPC (for a method that is not present in Golang, as such its exposure surface should be reduced). It replaces existing variables that are at the front.  (it's a method that was introduced not so long ago to accommodate `UtxoProcessor` in `kaspa-wallet-core`).  I have changed the way the versioning works in the RPC as such the previous data structure for this method no longer makes sense.